### PR TITLE
Provide exception messages that occurred during stop

### PR DESCRIPTION
**Pull request by doyleyoung**
Provide exception messages that occurred during stop, rather than generic message - related to #1251

Signed-off-by: Doyle Young <dyoung@gmail.com>

### CODE REVIEW

File: src/test/java/io/fabric8/maven/docker/service/RunServiceTest.java [hunk 1]
```diff
@@ -277,6 +279,38 @@ public void testWithException() throws Exception {
...
+           docker.stopContainer(secondName, 0); result = new DockerAccessException("TEST two");
+        }};
+
+        runService = new RunService(docker, queryService, tracker, logOutputSpecFactory, log);
+
+        try {
+            runService.stopStartedContainers(false, true, true, testLabel);
+            fail("Should have thrown exception");
+        } catch (DockerAccessException | ExecException e) {
+            assertEquals(e.getLocalizedMessage(), "(TEST two,TEST one)");
```
Comment (rohanKanojia) [inline src/test/java/io/fabric8/maven/docker/service/RunServiceTest.java:310]: nit, these arguments should be in opposite order: `assertEquals(expected, actual)`

File: src/test/java/io/fabric8/maven/docker/service/RunServiceTest.java [hunk 2]
```diff
@@ -277,6 +279,38 @@ public void testWithException() throws Exception {
...
+        LogOutputSpecFactory logOutputSpecFactory = new LogOutputSpecFactory(true, true, null);
+
+        new Expectations(){{
+           docker.stopContainer(firstName, 0); result = new DockerAccessException("TEST one");
+           docker.stopContainer(secondName, 0); result = new DockerAccessException("TEST two");
+        }};
+
+        runService = new RunService(docker, queryService, tracker, logOutputSpecFactory, log);
+
+        try {
```
Comment (rohanKanojia) [inline src/test/java/io/fabric8/maven/docker/service/RunServiceTest.java:306]: I think maybe we can improve this by using `assertThrows` method like this:
```java
Exception thrownException = assertThrows(DockerAccessException.class, () -> runService.stopStartedContainers(false, true, true, testLabel));
assertEquals("(TEST two,TEST one)", thrownException.getLocalizedMessage());
```
WDYT?

**Comment by rohanKanojia** [comment]
@doyleyoung: Thanks a lot for your PR :+1: . It looks good to be merged, I just have minor comment regarding the test.

**Review by rhuss — APPROVED** [review]
beside the minor nit, looks good to me. Thanks a lot !

**Comment by doyleyoung** [comment]
@rohanKanojia @rhuss  updated the test.  Thanks for the awesome project 👍

**Review by rohanKanojia — APPROVED** [review]

**Pull request merged**